taxi/views.py

```diff
@@ -87,3 +103,19 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
```

Comment:
In `DriverUpdateView`, ensure that the `DriverLicenseUpdateForm` correctly validates the `license_number` field according to the specified format. This is crucial for maintaining data integrity when updating driver information.